### DISCUSSION TITLE
Welcome to blog-template-using-nextjs-typescript-tailwindcss Discussions!

### DISCUSSION THREAD

**Discussion by webexpe13** [webexpe13/blog-template-using-nextjs-typescript-tailwindcss#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by shujaakbar2020** [comment]
I'm facing a build issue in this repository.
npm run build
![Screenshot from 2024-01-26 13-38-46](https://github.com/webexpe13/blog-template-using-nextjs-typescript-tailwindcss/assets/59687688/d8bd3407-cfce-4589-8776-87d3d81110a3)
